src/modules/app/actions/get-augur-node-network-id.test.js

```diff
@@ -12,36 +12,36 @@ describe("modules/app/actions/get-augur-node-network-id.js", () => {
     store.clearActions();
     __RewireAPI__.__ResetDependency__("augur");
   });
-  const test = t =>
-    it(t.description, done => {
+  const runTest = t =>
+    test(t.description, done => {
       store = configureMockStore([thunk])({ ...t.state });
       __RewireAPI__.__Rewire__("augur", t.stub.augur);
       store.dispatch(
         getAugurNodeNetworkId((err, augurNodeNetworkId) => {
-          t.assertions(err, augurNodeNetworkId, store.getActions());
+          t.expectations(err, augurNodeNetworkId, store.getActions());
           done();
         })
       );
     });
-  test({
+  runTest({
```

Comment:
These runTest helpers (previously, test) should be rewritten using jest constructs. beforeAll, beforeEach, etc....